.circleci/config.yml

```diff
@@ -133,6 +133,43 @@ jobs:
             make migrate
             make check
 
+  # ---- Front-end jobs ----
+  build-front:
+    docker:
+      - image: circleci/node:10
+    working_directory: ~/fun/src/frontend
+    steps:
+      - checkout:
+          path: ~/fun
+      - restore_cache:
+          keys:
+            - v1-front-dependencies-{{ checksum "yarn.lock" }}
+            - v1-front-dependencies-
+      - run:
+          name: Install frontend dependencies
+          command: make install
```

Comment:
There is no `Makefile` in `src/frontend` and no task `install` in the main `Makefile`. I suppose it should be `yarn install` and your should use the option `--frozen-lockfile`